app/routes/turbo.api.v8.artifacts.$hash.ts

```diff
@@ -87,7 +87,8 @@ export const action: ActionFunction = async ({ request, params, context }) => {
   }
 
   const storage = new CacheStorage();
-  const contentLength = Number.parseInt(request.headers.get('Content-Length') as string);
+
+  const contentLength = Number.parseInt((request.headers.get('Content-Length') as string) ?? 0);
```

Comment:
on http2 clients won't send the content-length header.

this will currently save the length as 0. This looks ugly in the ui, but it has no other negative effect.

it can be fixed later by updating the size after streaming the body into the store